packages/@pollyjs/adapter-fetch/tests/integration/adapter-test.js

```diff
@@ -21,3 +21,36 @@ describe('Integration | Fetch Adapter', function() {
   adapterTests();
   adapterBrowserTests();
 });
+
+describe('Integration | Fetch Adapter | Concurrency', function() {
```

Comment:
Can you also add a test against a null context and one without fetch?

Reply:
Yup need to add tests in general around context.  Will finish up tonight + docs